src/Crud/Controller/CrudControllerTrait.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace Biig\Melodiia\Crud\Controller;
+
+use Biig\Melodiia\Crud\CrudableModelInterface;
+
+trait ControllerTrait
+{
+    /**
+     * @param string $modelClass
+     * @return bool
+     */
+    public function isModelClassInvalid(string $modelClass): bool
```

Comment:
Naming it `assertModelClassInvalid ` and throw the exception inside would be great isn't it?